### PR TITLE
Task/attrtype namemappings

### PR DESCRIPTION
**Pull request by IvanHdzC**
Missing functional validation with @manucarrace

### CODE REVIEW

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericColumnAggregator.java [hunk 1]
```diff
@@ -128,6 +130,31 @@ public void aggregate(NGSIEvent event) {
                 aggregation.put(attrName + "_type", valuesType);
             } // if else
         } // for
+
+        if (event.getMappedCE() != null && isEnableNameMappings() && isEnableGeoParse()) {
+            mappedContextElement = event.getMappedCE();
+            mappedContextAttributes = mappedContextElement.getAttributes();
+            for (NotifyContextRequest.ContextAttribute mapedContextAttribute : mappedContextAttributes) {
+                if (mapedContextAttribute.getType() != null && (mapedContextAttribute.getType().equalsIgnoreCase("geo:json") || mapedContextAttribute.getType().equalsIgnoreCase("geo:point"))) {
```
Comment (fgalan) [inline cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericColumnAggregator.java:138]: The equalsIgnoreCase() remembers me to the hack we discarded at the end (PR https://github.com/telefonicaid/fiware-cygnus/pull/1877 PR https://github.com/telefonicaid/fiware-cygnus/pull/1880).

Should we use equal() ? Or this is intentional?

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [cygnus-ngsi][MySQLSink, PostgreSQLSink, PostgisSQLSink] Remove SQLBackend singleton, thus allowing two sinks of the same SQL-type in the same Cygnus instance
 [cygnus-ngsi][CKANSink] New datamodel for CKAN (dm-by-entity-id) implementing mapping: subservice -> org, entityId -> dataset, entityId -> resource (#1792)
 [cygnus-ngsi][PostgisSQLSink, PostgreSQLSink] Add dm-by-entity-database and dm-by-entity-database-schema.
-
+[cygnus-ngsi][PostgisSQLSink] Look for a "geo:something" attribute type on mapped context element of an event before generic aggregation
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:4]:
```suggestion
[cygnus-ngsi][PostgisSQLSink] Look for a "geo:json" or "geo:point" attribute type on mapped context element of an event before generic aggregation
```

Not sure but looking to the code modifications it seems "geo:something" only have these two possiblities.

File: cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericColumnAggregator.java [hunk 3]
```diff
@@ -128,6 +130,31 @@ public void aggregate(NGSIEvent event) {
                 aggregation.put(attrName + "_type", valuesType);
             } // if else
         } // for
+
+        if (event.getMappedCE() != null && isEnableNameMappings() && isEnableGeoParse()) {
```
Comment (mrutid) [inline cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericColumnAggregator.java:134]: We do this Just when isEnableNameMappings? I can't find where we do this for the base case.

**Comment by IvanHdzC** [comment]
Implemented on #1882.

**Pull request closed without merging**